Source/SwiftyCamViewController.swift

```diff
@@ -125,7 +125,14 @@ open class SwiftyCamViewController: UIViewController {
 
 	public var lowLightBoost                     = true
 
-	/// Set whether SwiftyCam should allow background audio from other applications if audio capture is permitted
+    /// Set whether to start by default on viewWillAppear
+    public var automaticallyStartsPreview        = true
+
+    /// Specifies the [videoGravity](https://developer.apple.com/reference/avfoundation/avcapturevideopreviewlayer/1386708-videogravity) for the preview layer.
+
+    public var videoGravity: PreviewVideoGravity = .resizeAspect
```

Comment:
#? can this be set after the camera is running? if not may want to tweak this api slightly to convey that more